include/vapi/vsl_int.h

```diff
@@ -91,6 +91,7 @@ enum VSL_tag_e {
 	SLT__Bogus = 0,
 #define SLTM(foo,flags,sdesc,ldesc)	SLT_##foo,
 #include "tbl/vsl_tags.h"
+	SLT__VXID = 253, /* XXX: does it belong after SLT__Reserved? */
```

Comment:
It belongs after SLT__Reserved. But I don't want to change the value of SLT__Reserved, as that will introduce compatibility issues. So I think you should avoid adding the VXID as a pseudo-header altogether, and use a flag in struct vex_lhs to show that the left hand side of the expression is the VXID of the trans, instead of a log record.